serving/ingress/cmd/controller/main.go

```diff
@@ -14,5 +15,11 @@ var ctors = []injection.ControllerConstructor{
 }
 
 func main() {
-	sharedmain.Main("openshift-ingress-controller", ctors...)
+	ctx := signals.NewContext()
+
+	// Disable leader election to allow both ingress controllers to do their job.
+	// TODO: Fix the respective clash in Knative's reconciler framework.
```

Comment:
do we track this somewhere upstream ? e.g. knaitve/pkg ?

Reply:
Yep https://github.com/knative/pkg/issues/2109